diff --git a/libqtile/confreader.py b/libqtile/confreader.py
index 1ba870b4..bda49a55 100644
--- a/libqtile/confreader.py
+++ b/libqtile/confreader.py
@@ -99,6 +99,25 @@ class Config:
                 value = getattr(self, key, default[key])
             setattr(self, key, value)

+    def reload_config_submodules(self, path):
+        """Reloads python files from same folder as config file."""
+        to_reload = []
+        folder = path.parent
+        for name, module in sys.modules.items():
+
+            # Skip built-ins and anything with no filepath.
+            if getattr(module, "__file__", False):
+                subpath = Path(module.__file__).parent
+
+                # Check if the module is in the config folder or subfolder
+                if folder in subpath.parents:
+
+                    # We can't reload in this loop as that will alter sys.modules
+                    to_reload.append(module)
+
+        for m in to_reload:
+            importlib.reload(m)
+
     def load(self):
         if not self.file_path:
             return
@@ -108,6 +127,7 @@ class Config:
         sys.path.insert(0, path.parent.as_posix())

         if name in sys.modules:
+            self.reload_config_submodules(path)
             config = importlib.reload(sys.modules[name])
         else:
             config = importlib.import_module(name)
