diff --git a/hardware-configuration.nix b/hardware-configuration.nix
index 5ade901..75733fe 100644
--- a/hardware-configuration.nix
+++ b/hardware-configuration.nix
@@ -4,29 +4,26 @@
 { config, lib, pkgs, modulesPath, ... }:
 
 {
-  imports =
-    [ (modulesPath + "/installer/scan/not-detected.nix")
-    ];
+  imports = [ ];
 
-  boot.initrd.availableKernelModules = [ "xhci_pci" "ahci" "usbhid" "sd_mod" ];
+  boot.initrd.availableKernelModules = [ "ata_piix" "mptspi" "uhci_hcd" "ehci_pci" "sd_mod" "sr_mod" ];
   boot.initrd.kernelModules = [ ];
-  boot.kernelModules = [ "kvm-intel" ];
+  boot.kernelModules = [ ];
   boot.extraModulePackages = [ ];
 
   fileSystems."/" =
-    { device = "/dev/disk/by-uuid/0c34058f-4469-4dbe-a367-abb8e5204a1f";
+    { device = "/dev/disk/by-uuid/1d403cbf-ec0f-4592-9b89-9a99528b7297";
       fsType = "ext4";
     };
 
   fileSystems."/boot/efi" =
-    { device = "/dev/disk/by-uuid/BC5D-A34F";
+    { device = "/dev/disk/by-uuid/52BC-D771";
       fsType = "vfat";
     };
 
   swapDevices =
-    [ { device = "/dev/disk/by-uuid/c3428fbf-ae2d-40f2-b4b8-c4d9f4bc0491"; }
+    [ { device = "/dev/disk/by-uuid/c91b195d-8f56-48d5-92cf-a374757e4fc7"; }
     ];
 
-  powerManagement.cpuFreqGovernor = lib.mkDefault "powersave";
   hardware.cpu.intel.updateMicrocode = lib.mkDefault config.hardware.enableRedistributableFirmware;
 }
diff --git a/home/config/xinitrc.nix b/home/config/xinitrc.nix
index 677c956..0f077c9 100644
--- a/home/config/xinitrc.nix
+++ b/home/config/xinitrc.nix
@@ -10,7 +10,10 @@
   xset -dpms s off
 
   # Compositor
-  picom &
+  # picom &
+
+  # Screensize
+  xrandr -s 1360x768
 
   # Start WM
   exec qtile start
diff --git a/system/default.nix b/system/default.nix
index b915e26..70a666b 100644
--- a/system/default.nix
+++ b/system/default.nix
@@ -5,7 +5,7 @@
     ../hardware-configuration.nix
     # ./gpu/nvidia.nix
     # ./gpu/amd.nix
-    ./gpu/intel.nix
+    # ./gpu/intel.nix
 
     ./bootloader.nix
     ./packages.nix
