diff --git a/home/config/xinitrc.nix b/home/config/xinitrc.nix
index 677c956..0f077c9 100644
--- a/home/config/xinitrc.nix
+++ b/home/config/xinitrc.nix
@@ -10,7 +10,10 @@
   xset -dpms s off
 
   # Compositor
-  picom &
+  # picom &
+
+  # Screensize
+  xrandr -s 1360x768
 
   # Start WM
   exec qtile start
diff --git a/system/default.nix b/system/default.nix
index b915e26..70a666b 100644
--- a/system/default.nix
+++ b/system/default.nix
@@ -5,7 +5,7 @@
     ../hardware-configuration.nix
     # ./gpu/nvidia.nix
     # ./gpu/amd.nix
-    ./gpu/intel.nix
+    # ./gpu/intel.nix
 
     ./bootloader.nix
     ./packages.nix
