diff --git a/home/config/xinitrc.nix b/home/config/xinitrc.nix
index 677c956..3e4b352 100644
--- a/home/config/xinitrc.nix
+++ b/home/config/xinitrc.nix
@@ -4,13 +4,16 @@
   #!/bin/sh
 
   # Keyboard layout
-  setxkbmap us -variant altgr-intl
+  setxkbmap latam
 
   # Screensaver
   xset -dpms s off
 
   # Compositor
-  picom &
+  # picom &
+
+  # Screensize
+  xrandr -s 1360x768
 
   # Start WM
   exec qtile start
diff --git a/system/bootloader.nix b/system/bootloader.nix
index 52ccaed..068ddcb 100644
--- a/system/bootloader.nix
+++ b/system/bootloader.nix
@@ -25,7 +25,7 @@
         enable = true;
         devices = [ "nodev" ];
         efiSupport = true;
-        useOSProber = true;
+        useOSProber = false;
         gfxmodeEfi = "1360x768";
       };
 
diff --git a/system/default.nix b/system/default.nix
index b915e26..70a666b 100644
--- a/system/default.nix
+++ b/system/default.nix
@@ -5,7 +5,7 @@
     ../hardware-configuration.nix
     # ./gpu/nvidia.nix
     # ./gpu/amd.nix
-    ./gpu/intel.nix
+    # ./gpu/intel.nix
 
     ./bootloader.nix
     ./packages.nix
